meetuppizza/settings/base.py

```diff
@@ -38,6 +38,7 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'super_user'
```

Comment:
Check out chapter 4 of Two Scoops. Does this app follow their guidelines? If it doesn't does it break the rules in a useful way? Is it too specific, too general, or just right?